typescript/ws-protocol/src/FoxgloveServer.ts

```diff
@@ -323,6 +331,8 @@ export default class FoxgloveServer {
       name: this.name,
       capabilities: this.capabilities,
       supportedEncodings: this.supportedEncodings,
+      metadata: this.metadata,
+      sessionId: this.sessionId,
```

Comment:
```suggestion
      sessionId: this.sessionId ?? `${new Date().toUTCString()}`,
```

:point_up: @jhurliman is this what you mean?

Reply:
Yeah either way, I just think having `sessionId` set automatically is a good behavior.